package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
-    "azure-storage": "^2.8.1",
+    "@azure/storage-blob": "^12.5.0",
```

Comment:
Minor: 12.6.0 has already been published, if you'd want to reference to the latest storage-blob pakcage.

Reply:
@EmmaZhu  I have updated `package.json`  with the latest storage-blob package

Reply:
I've approved the PR.